.github/workflows/pipeline.yml

```diff
@@ -37,11 +37,28 @@ jobs:
         CK8S_CLOUD_PROVIDER: ${{ matrix.cloud }}
         CI_CK8S_FLAVOR: ${{ matrix.flavor }}
         CK8S_PGP_FP: 529D964DE0BBD900C4A395DA09986C297F8B7757
-    - name: Run unit tests for OPA policies
-      uses: ./pipeline
-      with:
-        args: ./pipeline/test/services/unit-tests/test-opa.sh
```

Comment:
I sort of like the idea of actually having a script for this. The main reason is that later on if we add more unit tests (like the bat tests) we can gather them in a bigger script that you can run locally as well. I know that you can still run this command locally, but I guess it's a bit more hidden.
Since OPA has not been pushed lately it might be fine to have it like this for now and move it later again if we would like  that.

Reply:
Yea I don't mind. But I think we should move it from `./pipeline/test/services/unit-tests/test-opa.sh` to just `./pipeline/opa.sh` or something like that in that case, because the previous one is also a bit hidden.

Reply:
That is ok with me.
Alternatively put both the opa test and bat tests in some unit test folder and create a `.pipeline/unit-test.sh` that runs both. Then mention it in the main README and/or the public docs.

Reply:
We currently don't inform people on how to setup and run Bats locally other than through our pipeline Docker container. Let's cleanup the local testing workflow later! I've restored the test-opa script though.